week-1-practise/level-2/4-currency-conversion-2.js

```diff
@@ -7,17 +7,28 @@
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToUSD(price) {}
-function convertToBRL(price) {}
-
+function convertToUSD(price) {
+  return price * 1.4 + 0.01 * price;
```

Comment:
You could have written another function to simplify your coding.

```suggestion
function convertToUSD(price) {
  return convert(price, 1.4);
}

function convertToBRL(price) {
  return convert(price, 4.91);
}

function convert(price, exchangeRate) {
  return price * exchangeRate * 0.99;
}
```

Reply:
That's good way to do it,,, but mine doesn't mean it's wrong. Does it?

Reply:
Yours is correct as well.